tests/sentry/integrations/slack/test_notify_action.py

```diff
@@ -25,10 +25,7 @@ def setUp(self):
                 'bot_access_token': 'xoxb-xxxxxxxxx-xxxxxxxxxx-xxxxxxxxxxxx',
             }
         )
-        OrganizationIntegration.objects.create(
-            organization=event.project.organization,
-            integration=self.integration,
-        )
+        self.integration.add_organization(event.project.organization)
```

Comment:
oh oops approved before i looked at tests. i think this needs to be `event.project.organization_id` not the org

Reply:
oop, opened this to see if the tests even worked anyway. Thanks